### PR TITLE
update Hungarian translation

### PR DESCRIPTION
**Pull request by HuBandiT**
month and day names are normally lowercase in Hungarian. add month selection header format

### CODE REVIEW

**Comment by hebbet** [comment]
thank you merged

File: js/locales/bootstrap-datepicker.hu.js [hunk 1]
```diff
@@ -4,13 +4,15 @@
...
-		today: "Ma",
+		days: ["vasárnap", "hétfő", "kedd", "szerda", "csütörtök", "péntek", "szombat"],
+		daysShort: ["vas", "hét", "ked", "sze", "csü", "pén", "szo"],
+		daysMin: ["V", "H", "K", "Sze", "Cs", "P", "Szo"],
+		months: ["január", "február", "március", "április", "május", "június", "július", "augusztus", "szeptember", "október", "november", "december"],
+		monthsShort: ["jan", "feb", "már", "ápr", "máj", "jún", "júl", "aug", "sze", "okt", "nov", "dec"],
+		today: "ma",
 		weekStart: 1,
+		clear: "töröl",
+		/* headerFormat: "yyyy. MM", */ /* for month selection */
```
Comment (acrobat) [inline js/locales/bootstrap-datepicker.hu.js]: why is this line commented out?

Comment (hebbet) [inline js/locales/bootstrap-datepicker.hu.js]: will revert that line

Comment (HuBandiT) [inline js/locales/bootstrap-datepicker.hu.js]: Hungarian would need the year+month header to be in the specified 
format, however I could not find in the documentation the proper keyword 
to use for this.

So I just added that commented out line to indicate demand for such a 
setting, and provide the correct value should this setting already exist 
or get implemented in the future.

2015.09.02. 15:37 keltezéssel, Jeroen Thora írta:

Comment (hebbet) [inline js/locales/bootstrap-datepicker.hu.js]: done with 8b5dd026ed

**Pull request merged**